content/participate/index.html.haml

```diff
@@ -1,138 +1,196 @@
 ---
 layout: simplepage
-title: Participate and contribute
+title: Participate and Contribute
 section: participate
 ---
 
+:css
```

Comment:
Is there any way this css can go in a separate file?  `participate-cards.css` or something?